architecture-decision-records/0011-base25-encoding-for-consignment-reference.md

```diff
@@ -0,0 +1,88 @@
+# base25 encoding for consignment references
+
+**Date:** 2020-11-13
+
+## Context
+
+As part of the 'transfer summary' page, the user will see a 'consignment reference' to identify their consignment (which can be useful if a user needs to contact us about a specific consignment). We came to the decision that we liked the format:
```

Comment:
It would also be good to add the context that the reference is intended to be a human-readable reference. We already use UUIDs to identify consignments that go through TDR, but UUIDs are hard for users to recognise or type/read out.

I think they current plan is that we'll export both the TDR UUID and the consignment reference to the long-term preservation system, so we can use either ID to look up a consignment in the future.